src/generated/graphql.tsx

```diff
@@ -3920,6 +3978,77 @@ export function useOrganizationCardLazyQuery(
 export type OrganizationCardQueryHookResult = ReturnType<typeof useOrganizationCardQuery>;
 export type OrganizationCardLazyQueryHookResult = ReturnType<typeof useOrganizationCardLazyQuery>;
 export type OrganizationCardQueryResult = Apollo.QueryResult<OrganizationCardQuery, OrganizationCardQueryVariables>;
+export const ConfigDocument = gql`
+  query config {
+    configuration {
+      authentication {
+        enabled
+        providers {
+          name
+          label
+          icon
+          config {
+            __typename
+            ... on AadConfig {
```

Comment:
AadAuthProviderConfig, you need to update the schema against the latest definition